qt4/backintime-qt4_polkit

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ $(which pkexec) ]; then
+        pkexec --disable-internal-agent "/usr/bin/backintime-qt4" "$@"
```

Comment:
As I understand pkexec from man mages and other issues on the web, ```--disable-internal-agent``` helps when you're trying to execute a script with PolicyKit but you don't have a GUI (and thus no authentication agent).

However, it seems to me that one not execute the GUI for BIT (```backintime-qt4```) without a GUI already running.

Are you having difficulty getting prompted from the GUI because you don't have an authentication agent registered, or are having issues having BIT launch in root mode from a terminal (no GUI) because it won't prompt for a password (or some other situation?)